src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -158,7 +158,7 @@ protected AbstractEntityAIStructure(@NotNull final J job)
           /*
            * Check if we have to build something.
            */
-          new AITarget(IDLE, this::isThereAStructureToBuild, () -> START_BUILDING, 100),
+          new AITarget(IDLE, this::isThereAStructureToBuild, () -> START_BUILDING, 1),
```

Comment:
That's the wrong AI state you reset it =D

Reply:
I wonder why tho, i just pressed ctrl+z to undo the last change.